config/gulp/javascript.js

```diff
@@ -27,6 +27,31 @@ gulp.task('eslint', function (done) {
 
 });
 
+gulp.task('copy-jquery-javascript', function (done) {
+
+  dutil.logMessage(task, 'Copying JS from jQuery');
+
+  var stream = gulp.src('./node_modules/jquery/dist/jquery.min.*')
+    .pipe(gulp.dest('assets/js/vendor/jquery/'));
```

Comment:
Is there a reason this needs to live inside of a folder? Or can the file live in the `vendor` folder on it's own?